### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by HunterX-7**
In this Pull Request, I:

- Create the popup-window branch.
- Followed GitHub flow.
- Use HTML&CSS and JavaScript Linters.
- Write meaningful commit messages in the most important parts of the process.
- Check the following list and implement these features in the mobile menu:

  > When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
  > When the user clicks (or taps) the close (X) button, the popup disappears.
  > Use a JavaScript array to store all the information for all projects.
  > For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, and link to the source.
  > Update the main page so that the projects section is created dynamically using the information stored in that JavaScript object.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -10,3 +12,142 @@ function toggle() {
...
+                                            <li class="skills1">${obj[i].technologies[1]}</li>
+                                            <li class="skills1">${obj[i].technologies[2]}</li>
+                                            <li class="skills1">${obj[i].technologies[3]}</li>
+                                        </ul>
+                                        <p class="p3">${obj[i].description}</p>
+                                        <button class="button-interact antispacing1 display2"><a class="deco" href="${obj[i].live}" target="_blank">See Live <i
+                                                    class="i-back1"></i></a></button>
+                                        <button class="button-interact antispacing1 display2"><a class="deco" href="${obj[i].source}" target="_blank">See Source <i
+                                                    class="i-back2"></i></a></button>
+                                    </div>
```
Comment (ha-manel) [inline main.js]: - [x] Great job on implementing the modal window 👏 kindly try to follow the design as much as possible:
- [x] For the desktop version, the image should be bigger than it is right now, and the window is much bigger than the screen (I couldn't see the close button)

## Your design:
image

## Figma design:
image

File: main.js [hunk 2]
```diff
@@ -10,3 +12,142 @@ function toggle() {
...
+                                </div>
+                            </div>
+                    `;
+    main.appendChild(popup);
+    workSection.appendChild(main);
```
Comment (ha-manel) [inline main.js:144]: - [x] For the mobile version: the content of the popup isn't visible unless I zoom out, it's better if you make the popup scrollable so we can see all of the content 👍 

[ Optional ]
- [ ] To make the UX better, it's better if the body isn't scrollable when the modal is open.

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi Team👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

## Optional:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HunterX-7** [comment]
## Made the changes requested ✔️ 

- Redo the popup template to stick to the design as much as possible.
- Style the popup template according to the Figma design.
- Add the new template in the innerHTML of the loop.

File: main.js (hunk 2 above)
Comment (danmainah) [reply main.js:144]: - [ ]  @HunterX-7 great work implementing the changes requested, only a small portion remaining, in the comparison below you can see the live demo and the GitHub button different in placement, you are aligned to the right, while in the design they occupy the left and right side, kindly check it;


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image | image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @HunterX-7 , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HunterX-7** [comment]
## Made the changes requested ✔️ 

- Adjust the placement of the live demo and GitHub account to be aligned in between.
- Made more styling changes to the popup template to stick to the design as much as possible.


Detail popup window

**Review by mckent05 — APPROVED** [review]
Hi @HunterX-7,

# STATUS: APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approved


## Highlights
- Your popup window works as expected 👍 
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**